docs/tutorial.rst

```diff
@@ -306,17 +318,281 @@ trees. In this example we incorrectly inferred that 0 coalesces with the
+the `tskit <//tskit.readthedocs.io/>`_ library. The
+:ref:`tskit tutorial<sec_tutorial_stats>` provides much more detail. Below we just give a
+flavour of the possibilities.
+
+To quickly eyeball small datasets, we can draw the entire tree sequence, or
+:meth:`draw<tskit.Tree.draw>` the tree at any particular genomic position. The following
+code demonstrates how to use the :meth:`tskit.TreeSequence.at` method to obtain the tree
+1Mb from the start of the sequence, and plot it, colouring the tips according to
+population:
+
+.. code-block:: python
 
-.. todo:: Also add metadata and populations to this example, showing
-     how we retrieve the metadata from the tree sequence.
+    colours = {"Norway": "red", "France": "blue"}
+    colours_for_node = {}
+    for n in sparrow_ts.samples():
+        population_data = sparrow_ts.population(sparrow_ts.node(n).population)
+        colours_for_node[n] = colours[json.loads(population_data.metadata)['country']]
+    
+    individual_for_node = {}
+    for n in sparrow_ts.samples():
+        individual_data = sparrow_ts.individual(sparrow_ts.node(n).individual)
+        individual_for_node[n] = json.loads(individual_data.metadata)['name']
+    
+    sparrow_ts.at(1e6).draw(path="tree_at_1Mb.svg", height=700, width=1200,
+                            node_labels=individual_for_node,
+                            node_colours=colours_for_node)
+
+.. image:: _static/tree_at_1Mb.svg
+   :width: 100%
+   :alt: An example tree
+
+This tree seems to suggest that Norwegian and French individuals may not fall into
+discrete groups on the tree, but be part of a larger mixing population. Note, however,
+that this is only one of thousands of trees, and may not be typical of the genome as a
+whole. Additionally, most data sets will have far more samples than this example, so
+trees visualized in this way are likely to be huge and difficult to understand. It is
+possible to plot a subset of the tips by :meth:`simplifying<tskit.TreeSequence.simplify>`
+the tree sequence to a limited number of samples, but it is likely that most studies will
+instead rely on various statistical summaries of the trees. Storing genetic data as a
+tree sequence makes many of these calculations fast and efficient, and tskit has both a
+set of :ref:`commonly used methods<sec_stats>` and a framework that
+:ref:`generalizes population genetic statistics<sec_stats_general_api>`. For example,
+the allele or site frequency spectrum (SFS) can be calculated using
+:meth:`tskit.TreeSequence.allele_frequency_spectrum` and the allelic diversity ("Tajima's
+:math:`{\pi}`") using :meth:`tskit.TreeSequence.diversity`, both of which can also be
+calculated locally (e.g. :ref:`per tree or in genomic windows <sec_stats_windows>`). As
+a basic example, here's how to calculate genome-wide :math:`F_{st}` between the Norwegian
+and French (sub)populations:
+
+.. code-block:: python
+
+    sample_node_objs = [sparrow_ts.node(n) for n in sparrow_ts.samples()]
```

Comment:
Simpler:
```
samples_by_pop = [sparrow_ts.samples(population=pop_id) for pop_id in range(sparrow_ts.num_populations)]
```

Reply:
Of course, I forgot you could provide a `population` param to `samples()`. But I do use the list of node objects later, in e.g.

`sample_pops = [json.loads(sparrow_ts.population(n.population).metadata)['country'] for n in sample_node_objs]`

I'll need to rework these.